functions-methods.js

```diff
@@ -20,9 +23,24 @@
 // typeOfEmail("novi.nlaapjesk@outlook.com") geeft geeft "Extern" <-- deze moet het ook doen!
 // typeOfEmail("a.wiersma@outlook.com") geeft "Extern"
 
+function typeOfEmail(str) {
+    const sRegex = RegExp("@novi-education");
+    const mRegex= RegExp("@novi");
+    if(sRegex.test(str) === true){
+        return "student";
+    } else if(mRegex.test(str) === true) {
```

Comment:
Kan ook iets versimpeld worden:
```suggestion
    if(sRegex.test(str)){
        return "student";
    } else if(mRegex.test(str)) {
```